l10n/en/mozorg/about.ftl

```diff
@@ -0,0 +1,65 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+### URL: https://www-dev.allizom.org/about/
+
+# HTML Page title
+about-learn-about-mozilla = Learn About { -brand-name-mozilla }
+
+# HTML page description
+about-mozilla-makes-browsers-apps-desc = { -brand-name-mozilla } makes browsers, apps, code and tools that put people before profit. Our mission: Keep the internet open and accessible to all.
+
+# Main heading
+about-mozilla-makes-browsers-apps = { -brand-name-mozilla } makes browsers, apps, code and tools that put people before profit.
+
+# Main sub-heading
+about-our-mission-keep-the-internet = Our mission: Keep the internet open and accessible to all.
+
+# Main call to action
+about-read-our-mission = Read Our Mission
+
+about-our-mission-in-action = Our Mission in Action
+about-pioneers-of-the-open-web = Pioneers of The Open Web
+about-our-leadership-has-been-at = Our leadership has been at the forefront of building a healthier internet since Day 1. What began as an alternative to corporate domination has grown into a global force for good online.
+about-firefox-fast-for-good = { -brand-name-firefox }: Fast for Good
+about-when-you-use-the-new-firefox = When you use the new { -brand-name-firefox }, you get a blazing fast experience while supporting { -brand-name-mozilla }’s mission to keep the internet healthy, weird and welcoming to all.
+about-walking-our-privacy-talk = Walking Our Privacy Talk
+about-when-the-facebook-breach-was = When the { -brand-name-facebook } breach was revealed, { -brand-name-mozilla } had an immediate response – and a { -brand-name-firefox } product to support user privacy.
+about-talking-internet-issues-irl = Talking Internet Issues IRL
+about-in-mozillas-irl-podcast-host = In { -brand-name-mozilla }’s IRL podcast, host Manoush Zomorodi shares real stories of life online and real talk about the future of the Web.
+about-corporation-foundation-not = Corporation. Foundation. Not-for-profit.
+about-learn-about-the-mozilla-foundation = Learn about the { -brand-name-mozilla-foundation }
+about-mozilla-puts-people-over-profit = { -brand-name-mozilla } puts people over profit in everything we say, build and do. In fact, there’s a non-profit Foundation at the heart of our enterprise.
+about-the-mozilla-manifesto = The { -brand-name-mozilla } Manifesto
+about-the-principles-we-wrote-in = The principles we wrote in 1998 still guide us today. And in 2018, we created an addendum to emphasize inclusion, privacy and safety for everyone online.
+about-read-the-manifesto = Read The Manifesto
+about-a-global-view = A Global View
+about-with-offices-all-over-the = With <a href="{ $url }">offices all over the world</a>, we consider the internet from multiple cultures and contexts.
```

Comment:
Should we add a comment for this variable? I'm not even sure those comments are all that useful to localizers, maybe just a nice-to-have.

Reply:
Ugh, I missed this one. The L10n files are already in Pontoon, so let's fix this one later next time the file is updated 👍